autoconfigure/storage-applicationinsights/src/main/resources/zipkin-server-applicationinsights.yml

```diff
@@ -0,0 +1,7 @@
+# When enabled, this allows shorter env properties (ex -Dspring.profiles.active=applicationinsights)
+zipkin:
+  storage:
+    applicationinsights:
+      instrumentationKey: ${AI_INSTRUMENTATION_KEY:}
```

Comment:
I will change it to this... i just thought AI_IN... will make the key shorter..and in power shell I often type so less work :)